docker/nginx/nginx.wsl.conf

```diff
@@ -0,0 +1,76 @@
+user nginx;
+
+worker_processes  2;
+
+pid /var/run/nginx.pid;
+
+error_log  /var/log/nginx.error_log  info;
+
+events {
+    worker_connections   2000;
+}
+
+http {
+    log_format main      '$remote_addr - $remote_user [$time_local] '
+                         '"$request" $status $bytes_sent '
+                         '"$http_referer" "$http_user_agent" '
+                         '"$gzip_ratio"';
+
+    log_format download  '$remote_addr - $remote_user [$time_local] '
+                         '"$request" $status $bytes_sent '
+                         '"$http_referer" "$http_user_agent" '
+                         '"$http_range" "$sent_http_content_range"';
+
+    client_header_timeout  3m;
+    client_body_timeout    3m;
+    send_timeout           3m;
+
+    client_header_buffer_size    1k;
+    large_client_header_buffers  4 4k;
+
+    gzip on;
+    gzip_min_length  1100;
+    gzip_buffers     4 8k;
+    gzip_types       text/plain;
+
+    output_buffers   1 32k;
+    postpone_output  1460;
+
+    sendfile         on;
+    tcp_nopush       on;
+    tcp_nodelay      on;
+
+    keepalive_timeout  75 20;
+
+    server {
+        listen        80;
+        listen        443 ssl default_server;
+        server_name   _;
+        ssl_certificate     /etc/nginx/cert/localhost.crt;
+        ssl_certificate_key /etc/nginx/cert/localhost.key;
+        ssl_session_timeout  		5m;
+        ssl_protocols  			TLSv1.2 TLSv1.3;
+        ssl_ciphers         ECDHE-RSA-AES256-GCM-SHA512:DHE-RSA-AES256-GCM-SHA512:ECDHE-RSA-AES256-GCM-SHA384:DHE-RSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-SHA384:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384;
+        ssl_prefer_server_ciphers   	on;
+
+        access_log   /var/log/nginx.access_log  main;
+
+        location / {
+            proxy_pass         http://127.0.0.1:3000/;
```

Comment:
Want to give that try and see if you can delete this duplicate one?

Reply:
Deleted the `nginx.wsl.conf` file and tried using the original one with `127.0.0.1` instead of `frontend` and `backend` and it worked. But I'm suspicious as to if this would now work on other docker clients or does this break it?